recipe/meta.yaml

```diff
@@ -122,7 +124,7 @@ outputs:
         # cmake
         - cd cf_test
         - cmake $CMAKE_ARGS .   # [build_platform == target_platform and unix]
-        - cmake %CMAKE_ARGS% .  # [win]
+        - cmake %CMAKE_ARGS% .  # [build_platform == target_platform and win]
```

Comment:
I don't really understand why this condition was added

Reply:
Will revert